charts/argo/crds/cluster-workflow-template-crd.yaml

```diff
@@ -11,7 +11,13 @@ spec:
   scope: Cluster
   names:
     kind: ClusterWorkflowTemplate
+    listKind: ClusterWorkflowTemplateList
     plural: clusterworkflowtemplates
     shortNames:
-    - clusterwftmpl
```

Comment:
Any reason for the formatting changes here and in other places? looking at the original CRDs in Argo they don't indent arrays either, could we keep these the same?

Reply:
reason: vscode auto formatter ;) 
Fixed it - sorry about that.